torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -3577,11 +3579,21 @@ struct to_ir {
         if (input_size == 2) {
           start_index = emitSugaredExpr(inputs[1], 1)->asValue(loc, method);
         }
-
-        if (input_size > 2) {
+        auto arg_size = input_size + attribute_size;
+        if (arg_size > 2) {
           throw ErrorReport(loc)
-              << "enumerate expected at most 2 arguments, got " << input_size;
+              << "enumerate expected at most 2 arguments, got " << arg_size;
         }
+
+        if (attribute_size == 1) {
+          if (attributes[0].name().name() != "start") {
+            throw ErrorReport(loc)
+                << "enumerate expected kwarg name 'start', got '"
+                << attributes[0].name().name() << "'";
+          }
+          start_index = emitSugaredExpr(attributes[0].value(), 1)->asValue(loc, method);
+        }
```

Comment:
so is it true that `attributes` just means `kwargs`?

Reply:
yes, I think so. in the same file, we have https://github.com/pytorch/pytorch/blob/c71886e04827f9fa87a3a4a0ee4e76f3b1dc22e1/torch/csrc/jit/frontend/ir_emitter.cpp#L3350